modin/pandas/test/test_groupby.py

```diff
@@ -1436,7 +1440,13 @@ def test_groupby_with_kwarg_dropna(groupby_kwargs, dropna):
         and any(col in modin_df.columns for col in by_kwarg)
     ):
         df_equals(md_grp.sum(), pd_grp.sum())
-        df_equals(md_grp.size(), pd_grp.size())
+        if not (
+            groupby_kwargs == {"level": 1, "axis": 1}
+            and PandasCompatVersion.CURRENT == PandasCompatVersion.LATEST
+        ):
+            # skip the check for df.groupby().size() on pandas 1.5.0
+            # as it's failing due to pandas bug: https://github.com/pandas-dev/pandas/issues/48738
+            df_equals(md_grp.size(), pd_grp.size())
```

Comment:
Should we fix this as part of a separate issue?

Reply:
Yes, I was planning on making a tracker for that, but haven't made one yet. Will do.

Reply:
Please put it here once created.

Reply:
replaced pandas tracker with Modin one